src/app/organization/working-days/working-days.component.html

```diff
@@ -0,0 +1,38 @@
+<div class="container">
+  <mat-card>
+
+    <div fxLayout="row" fxLayout.lt-md="column">
+      <h3 fxFlex="20%">Working Days</h3>
+      <div fxFlex="80%" fxLayout="column" class="input checkboxes" *ngFor="let day of weekDays; let i = index;">
+        <mat-checkbox [(ngModel)]="weekDays[i].checked">{{day.value}}</mat-checkbox>
+      </div>
+    </div>
+
+    <div fxLayout="row" fxLayout.lt-md="column">
+      <h3 fxFlex="20%">Payments due on non working days</h3>
```

Comment:
To keep consistency with other components keep this as a label instead of the values that you have mentioned as label below: `Payments due on non-working days`